paperqa/core.py

```diff
@@ -38,7 +40,8 @@ def escape_newlines(match: re.Match) -> str:
 async def map_fxn_summary(
     text: Text,
     question: str,
-    prompt_runner: PromptRunner | None,
+    summary_llm_model: LLMModel | None,
+    prompt_details: tuple[str, str] | None,
```

Comment:
I don't like the name "prompt details, really this is two prompt templates.

Can you make the name `prompt_templates` or something more explicit